source/content/plugins-known-issues.md

```diff
@@ -249,9 +249,31 @@ if (isset($_ENV['PANTHEON_ENVIRONMENT'])) {
 
 For more details, see [SERVER_NAME and SERVER_PORT on Pantheon](/server_name-and-server_port).
 
-**Issue 2:** Local file attachments set in the admin panel cannot come from the `uploads` folder. As described in [this plugin issue](https://wordpress.org/support/topic/local-file-attachments-do-not-work-in-pantheon-hosting/), the plugin code fails for upload directories that are symlinks.
+**Issue 2:** In order to attach or upload files, local file attachments set in the admin panel cannot come from the `uploads` folder. Therefore, you must direct attachments to a temporary folder.
 
-**Solution:** Until the plugin is updated to allow symlink paths, you can commit your local attachment files to the codebase in `wp-content` or another subdirectory thereof.
+**Solution:** You can customize the upload path for the temporary folder using the following:
+
+`define( 'WPCF7_UPLOADS_TMP_DIR', '/your/file/path' );`
```

Comment:
Because the thread suggests using the path, it would be best to use the WordPress constant `WP_CONTENT_DIR` when defining that. So it would be:

```
define( 'WPCF7_UPLOADS_TMP_DIR',  WP_CONTENT_DIR . '/uploads/wpcf7_uploads' );
```

The other takeaway on that being that the folder would need to reside in the environment (dev,test,live)

Reply:
Thanks Jose 🙂